frontend/ui/widget/touchmenu.lua

```diff
@@ -961,4 +963,206 @@ function TouchMenu:onBack()
     self:backToUpperMenu()
 end
 
+------ the menu search functionality
+function TouchMenu:search(search_for)
+    local foundMenuItem = {}
```

Comment:
plural, `foundMenuItems` - or `found_menu_items` (as camelCase is for methods) - or just `results`.